src/grading-settings/GradingSettings.jsx

```diff
@@ -156,6 +160,7 @@ const GradingSettings = ({ intl, courseId }) => {
                       resetDataRef={resetDataRef}
                       setOverrideInternetConnectionAlert={setOverrideInternetConnectionAlert}
                       setEligibleGrade={setEligibleGrade}
+                      defaultGradeDesignations={gradingSettings.defaultGradeDesignations}
```

Comment:
@xitij2000 When trying to test the changes locally I'm getting an error on this line, since `gradingSettings` doesn't seem to be defined.

Reply:
Surprisingly I never faced this issue, but I can understand why it'd happen. Hopefully the latest commits fix this.